articles/modules/ROOT/pages/using-subqueries-to-avoid-the-eager.adoc

```diff
@@ -0,0 +1,130 @@
+= Using Subqueries to Avoid the Eager
+
+:slug: using-subqueries-to-avoid-the-eager
+:author: Andrew Bowman
+:neo4j-versions: 4.1, 3.5
```

Comment:
Version 4.1 does not exist on WordPress, should we create it? I can do it but I need a confirmation from the KB team.

Reply:
Sorry, was that created? We will need entries for each minor version, so we will need 4.1 and 4.2.